index.js

```diff
@@ -12,6 +12,7 @@ const normalize = require('array-normalize')
 const { float32, fract32 } = require('to-float32')
 const WeakMap = require('es6-weak-map')
 const parseRect = require('parse-rect')
+const findIndex = require('array.prototype.findindex');
```

Comment:
That's 5kb dependency https://bundlephobia.com/result?p=array.prototype.findindex@2.1.0, I'd propose not polyfilling but directly use `findIndex`

Reply:
Without the polyfill it fails the image test in plotly.
How about https://bundlephobia.com/result?p=array-find-index@1.0.2? it's only 287B

Reply:
Yes, this one is better